src/System.Linq.Expressions/src/System/Linq/Expressions/IndexExpression.cs

```diff
@@ -437,14 +448,14 @@ private static void ValidateAccessor(Expression instance, MethodInfo method, Par
             ContractUtils.RequiresNotNull(arguments, nameof(arguments));
 
             ValidateMethodInfo(method, nameof(method));
-            if ((method.CallingConvention & CallingConventions.VarArgs) != 0) throw Error.AccessorsCannotHaveVarArgs(nameof(method));
+            if ((method.CallingConvention & CallingConventions.VarArgs) != 0) throw Error.AccessorsCannotHaveVarArgs(paramName);
```

Comment:
That could be done for a lot of cases throughout the file, and indeed many like it. I think I'll keep the change made the minimal to see what it is I'm doing here for now, if nobody objects. A PR that just changes braces and similar stylistic matters would be good, though likely to be conflicting with a lot of other PRs right now.